cmd/agent/Dockerfile

```diff
@@ -22,6 +22,10 @@ RUN apt-get update && apt-get install -t bullseye-backports -qy libsystemd-dev &
   apt-get install -qy tzdata ca-certificates && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Add support for bcc bindings required to run the eBPF integration
+RUN echo "deb [trusted=yes] https://repo.iovisor.org/apt/bionic bionic-nightly main" | tee /etc/apt/sources.list.d/iovisor.list
+RUN apt-get update && apt-get install -qy libbpfcc-dev
```

Comment:
The `-dev` package (headers) isn't needed at runtime, suggest installing the library package here `libbpfcc`.